providers/tests/openlineage/plugins/test_adapter.py

```diff
@@ -585,9 +586,8 @@ def test_emit_dag_started_event(mock_stats_incr, mock_stats_timer, generate_stat
             "dag_id": "dag_id",
             "data_interval_start": event_time.isoformat(),
             "data_interval_end": event_time.isoformat(),
-            "external_trigger": False if AIRFLOW_V_3_0_PLUS else None,
             "run_id": run_id,
-            "run_type": None,
+            "run_type": DagRunType.MANUAL if AIRFLOW_V_3_0_PLUS else None,
```

Comment:
```suggestion
            "run_type": DagRunType.MANUAL,
```

run_type has been around for _a long time_, we don't need this switch I don't think?